tests/contract-playground/src/AderynIgnoreCustomDetectors.sol

```diff
@@ -0,0 +1,30 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.20;
```

Comment:
```suggestion
pragma solidity 0.8.19;
```

Reply:
Any reason for this ? I'm missing context ..

Reply:
✅